### DISCUSSION TITLE
Is there a way to remove the create, save, delete buttons?

### DISCUSSION THREAD

**Discussion by codedge** [filamentphp/filament#377] [Help] [answered]
For a resource we use import jobs to fill the database and just use Filament to show the imported records and do some sorting. Is there a way to remove the "Create" button in the list view and the "Delete" and "Save" button in the edit view as we only want the records be managed via the import job / API?

**Comment by danharrin** [comment, marked as answer]
Yes, you may create a custom page, where you can render a table in the same way that the `ListRecords` base class does.
